CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: How to Contribute
+---
+Software Carpentry is an open source project,
+and we welcome contributions of all kinds:
+new lessons,
+fixes to existing material,
+bug reports,
+and reviews of proposed changes are all equally welcome.
+
+By contributing,
+you are agreeing that Software Carpentry may redistribute your work under
+[these licenses](LICENSE.md).
```

Comment:
I'm OK with this line be

~~~
[these licenses](LICENSE.md).
~~~

To clarify my first comment, if `CONTRIBUTING.html` was available the link should point to `LICENSE.html` that is already available because is probably the file that the user want. Since we are going to keep only `CONTRIBUTING.md` I'm +1 to link against `LICENSE.md`.

@wking Pandoc doesn't change the links:

~~~
$ pandoc -f markdown -t html <<EOF
[license](LICENSE.md)
EOF
<p><a href="LICENSE.md">license</a></p>
~~~

This is the same behavior of others Markdown to HTML:

~~~
$ kramdown <<EOF
[license](LICENSE.md)
EOF
<p><a href="LICENSE.md">license</a></p>
~~~